libs/dbr/context.go

```diff
@@ -0,0 +1,42 @@
+package dbr
+
+import "context"
+
+// key is a private type to prevent collisions with other packages.
```

Comment:
Thanks, clarified with this:
```
// key is a package-local type to use for context keys.
//
// Using an unexported type for context keys prevents key collisions across
// packages since external packages cannot create values of this type.
```